common/lib/protocol-definitions/src/clients.ts

```diff
@@ -34,15 +84,30 @@ export interface IClient {
 	timestamp?: number;
 }
 
+/**
+ * TODO
+ */
 export interface ISequencedClient {
+	/**
+	 * The underlying client details.
+	 */
 	client: IClient;
 
+	/**
+	 * TODO
```

Comment:
the sequence number of the client's join message/op; when the client joined

Reply:
Thanks!!